chime_utils/dgen/mixer6.py

```diff
@@ -390,6 +390,21 @@ def check_not_old_dev(ann_jsons):
         raise FileNotFoundError
 
 
+def adjust_eval_annotation(ann):
```

Comment:
eval annotation for mixer 6 in the provided JSONs does not correspond with provided audios